File: iniciante/python/1074-par-ou-impar.py
def entrada(numero):
    n = list()
    for num in range(numero):
        n.append(int(input()))
    return n


def par_ou_impar(valores):
    for valor in valores:
        if valor == 0:
            print('NULL')
        else:
            if valor % 2 == 0:
                print('EVEN', end=' ')
            else:
                print('ODD', end=' ')
            if valor > 0:
                print('POSITIVE')
            else:
                print('NEGATIVE')


number = entrada(int(input()))
par_ou_impar(number)

File: iniciante/c/1074-par-ou-impar.c
#include <stdio.h>

int main(){
    int n_valores, i, valor;
    scanf("%d", &n_valores);
    for (i = 0; i < n_valores; i++){
        scanf("%d", &valor);
        if (valor == 0){
            printf("NULL\n");
        }
        else{
            if (valor % 2 == 0){
                printf("EVEN ");
            }
            else{
                printf("ODD ");
            }
            if (valor > 0){
                printf("POSITIVE\n");
            }
            else{
                printf("NEGATIVE\n");
            }
        }
    }
    return 0;
}
